[2025-01-06T19:18:21.521+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-06T19:18:21.578+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kaggle_dag.start_task manual__2025-01-06T19:18:16.983417+00:00 [queued]>
[2025-01-06T19:18:21.614+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kaggle_dag.start_task manual__2025-01-06T19:18:16.983417+00:00 [queued]>
[2025-01-06T19:18:21.616+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-06T19:18:21.668+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): start_task> on 2025-01-06 19:18:16.983417+00:00
[2025-01-06T19:18:21.687+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=98) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-06T19:18:21.691+0000] {standard_task_runner.py:72} INFO - Started process 99 to run task
[2025-01-06T19:18:21.687+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'kaggle_dag', 'start_task', 'manual__2025-01-06T19:18:16.983417+00:00', '--job-id', '857', '--raw', '--subdir', 'DAGS_FOLDER/kaggle_dag.py', '--cfg-path', '/tmp/tmp902ujxew']
[2025-01-06T19:18:21.693+0000] {standard_task_runner.py:105} INFO - Job 857: Subtask start_task
[2025-01-06T19:18:21.839+0000] {task_command.py:467} INFO - Running <TaskInstance: kaggle_dag.start_task manual__2025-01-06T19:18:16.983417+00:00 [running]> on host b5c0ace8ab2f
[2025-01-06T19:18:22.168+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='kaggle_dag' AIRFLOW_CTX_TASK_ID='start_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-06T19:18:16.983417+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-06T19:18:16.983417+00:00'
[2025-01-06T19:18:22.170+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-06T19:18:22.279+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-01-06T19:18:22.281+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Here we start! "']
[2025-01-06T19:18:22.301+0000] {subprocess.py:99} INFO - Output:
[2025-01-06T19:18:22.311+0000] {subprocess.py:106} INFO - Here we start!
[2025-01-06T19:18:22.312+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-06T19:18:22.380+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-06T19:18:22.381+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=kaggle_dag, task_id=start_task, run_id=manual__2025-01-06T19:18:16.983417+00:00, execution_date=20250106T191816, start_date=20250106T191821, end_date=20250106T191822
[2025-01-06T19:18:22.439+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-06T19:18:22.518+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-06T19:18:22.525+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
